proto/dydxprotocol/listing/tx.proto

```diff
@@ -1,7 +1,31 @@
 syntax = "proto3";
 package dydxprotocol.listing;
 
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+
 option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/listing/types";
 
 // Msg defines the Msg service.
-service Msg {}
\ No newline at end of file
+service Msg {
+  // EnablePermissionlessMarketListing enables/disables permissionless market
+  // listing
+  rpc EnablePermissionlessMarketListing(MsgEnablePermissionlessMarketListing)
+      returns (MsgEnablePermissionlessMarketListingResponse);
+}
+
+// MsgEnablePermissionlessMarketListing is used to enable/disable permissionless
+// market listing
+message MsgEnablePermissionlessMarketListing {
```

Comment:
mega nit: would `MsgSetEnable...` make more sense given that it's used to enable and disable?